make-util.js

```diff
@@ -41,12 +52,12 @@ var ensureExists = function (checkPath) {
 
     if (!exists) {
         fail(checkPath + ' does not exist');
-    }    
+    }
```

Comment:
i think i hit alt+shift+f on the file which formatted everything. i didn't realize it until later.